app/views/listings/index.html.erb

```diff
@@ -1,2 +1,18 @@
 <h1>Listings#index</h1>
 <p>Find me in app/views/listings/index.html.erb</p>
+
+<div class="listings-container">
+  <% @listings.each do |listing| %>
+    <div class="listing">
+      <h2>
+        <%= link_to listing.title, listing %>
```

Comment:
The listing path here is incomplete - where do you want this link to link to?

Reply:
Also thanks :)